qmk_storage.py

```diff
@@ -41,17 +41,17 @@
     endpoint_url=S3_HOST,
     aws_access_key_id=S3_ACCESS_KEY,
     aws_secret_access_key=S3_SECRET_KEY,
-    config=botocore.client.Config(signature_version='s3'),
+    config=botocore.config.Config(signature_version='s3v4')
 )
 
 # Check to see if S3 is working, and if not print an error in the log.
 for bucket in [S3_BUCKET, COMPILE_S3_BUCKET]:
     try:
-        s3.create_bucket(Bucket=bucket)
+        s3.get_bucket_location(Bucket=bucket)
```

Comment:
`create_bucket` is needed for [qmk_web_stack](https://github.com/qmk/qmk_web_stack).
```suggestion
        s3.create_bucket(Bucket=bucket)
```